packages/mdc-slider/foundation.js

```diff
@@ -414,6 +471,237 @@ class MDCSliderFoundation extends MDCFoundation {
+      const leftTopNeckCornerCenterY = Math.sqrt(Math.pow(topLobeRadius+topNeckRadius, 2) -
+        Math.pow(15 - (topLobeHorizontalRight)/2, 2));
+
+      pointA.x = rightStemX + (topNeckRadius - (Math.cos(leftTopNeckCornerTheta) * topNeckRadius)) + offsetX;
+      pointA.y =
+        topLobeRadius + (leftTopNeckCornerCenterY - (Math.sin(leftTopNeckCornerTheta) * topNeckRadius)) + offsetY,
+      pointB.y = topLobeRadius + leftTopNeckCornerCenterY + offsetY;
+    }
+
+    // Path is created with string concatenation
+    let path = 'M ' + start.x + ' ' + start.y
+      + ' A ' + topLobeRadius + ' ' + topLobeRadius + ' 0 0 1 ' + pointA.x + ' ' + pointA.y;
+    // If there is extra right that needs to be added, it is added here
+    if (extraHorizontalWidth > 0 && extraHorizontalWidthRight > 0) {
+      path = path + ' L ' + (pointA.x - extraHorizontalWidthRight) + ' ' + pointA.y;
+    }
+    // The path continues to be concatenated here
+    path = path + ' A ' + topNeckRadius + ' ' + topNeckRadius + ' 0 0 0 ' + pointB.x + ' ' + pointB.y
+      + ' L ' + pointC.x + ' ' + pointC.y
+      + ' A ' + bottomNeckRadius + ' ' + bottomNeckRadius + ' 0 0 0 ' + pointD.x + ' ' + pointD.y
+      + ' A ' + bottomLobeRadius + ' ' + bottomLobeRadius + ' 0 0 1 ' + pointE.x + ' ' + pointE.y
+      + ' A ' + bottomLobeRadius + ' ' + bottomLobeRadius + ' 0 0 1 ' + pointF.x + ' ' + pointF.y
+      + ' A ' + bottomNeckRadius + ' ' + bottomNeckRadius + ' 0 0 0 ' + pointG.x + ' ' + pointG.y
+      + ' L ' + pointH.x + ' ' + pointH.y
+      + ' A ' + topNeckRadius + ' ' + topNeckRadius + ' 0 0 0 ' + pointI.x + ' ' + pointI.y;
+    // If there is extra left that needs to be added, it is added here
+    if (extraHorizontalWidth > 0 && extraHorizontalWidthLeft > 0) {
+      path = path + ' L ' + (pointI.x - extraHorizontalWidthLeft) + ' ' + pointI.y;
+    }
+    // Path is finished off here
+    path = path + ' A ' + topLobeRadius + ' ' + topLobeRadius + ' 0 0 1 ' + end.x + ' ' + end.y + ' Z';
+    return path;
+  }
+
+  /**
+   * Calculates the value label text x attribute value
+   * @param {number} translatePx
+   * @return {number}
+   */
+  calcValueLabelTextXValue_(translatePx) {
+    const localeStringWidth = this.calcLocaleStringWidth_();
+    let extraTranslateValue = 0;
+    let topLobeHorizontal = 0;
+    const digitWidth = this.adapter_.getDigitWidth();
+
+    // Calculates default translateX value for the size of the text
+    let xValue = (34 - localeStringWidth);
+    if (this.value_.toString().length > 5) {
+      xValue = (xValue * 0.75) + 4;
+    } else {
+      xValue = xValue / 2;
+    }
+    if (this.adapter_.isRTL()) {
+      xValue = xValue + localeStringWidth;
+    }
+
+    // Calculates any extra translate value on either side if the slider is near the ends
+    if (this.value_.toString().length > 2) {
+      topLobeHorizontal = localeStringWidth - (2 * digitWidth);
+    }
+    const extra = topLobeHorizontal - 30;
```

Comment:
Update these constants to use the new names `extraHorizontalWidth`, `extraHorizontalWidthLeft`, `extraHorizontalWidthRight`

Reply:
Got it!